internal/service/service.go

```diff
@@ -295,11 +295,11 @@ func getSetProfileParams(owner sdk.AccAddress, p schema.V1Profile) *storage.SetP
 }
 
 func getPDVOwnerPrefix(owner string) string {
-	return fmt.Sprintf("pdv/%s", owner)
+	return fmt.Sprintf("%s/pdv", owner)
```

Comment:
Breaking change right? We need to keep in mind as far as going to crease some dataset samples.